int32 id
geometry_msgs/Pose pose
---
int32[] ids

